commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ftp/FtpClient.java

```diff
@@ -69,5 +69,14 @@ default void setBufferSize(final int bufferSize) throws FileSystemException {
     }
 
     OutputStream storeFileStream(String relPath) throws IOException;
-
+    
+    default boolean features() throws IOException {

```

Comment:
I don't get this method name.

Reply:
It's named the same as the delegate FTPClient method. I've renamed to `getRemoteFtpFeatures`